README.md

```diff
@@ -140,7 +140,7 @@ with the default vehicle prefix: `saic/<saic_user>/vehicles/<vehicle_id>`
 | /doors/locked/set                        | true/false                                           | Lock or unlock your car. This is not always working. It might take some time until it takes effect. Don't trust this feature. Use your car key!                                                                                       |
 | /climate/remoteTemperature/set           | temperature                                          | Set A/C temperature                                                                                                                                                                                                                   |
 | /climate/remoteClimateState/set          | on/off/front/blowingonly                             | Turn A/C on or off, activate A/C blowing (front) or blowing only (blowingonly)                                                                                                                                                        |
-| /climate/heatedSeatsFrontLeftLevel/set   | 0-3 or 0-1 depending on model                        | Set heated seats level for the front left seat. Some cars have three levels while others just an on-off switch. 0 means OFF                                                                                                           |
+| /climate/heatedSeatsFrontLeftLevel/set   | 0-3 or 0-1 depending on model                        | Set heated seats level for the front left seat. Some cars havedefault three levels while others just an on-off switch. 0 means OFF                                                                                                    |
```

Comment:
typo?

Reply:
Yes, sorry! PyCharms warnings and auto formatting confused me a bit